### PR TITLE
BREAKING CHANGE: remove opt.ws

### PR DESCRIPTION
**Pull request by saschanaz**
It has never been documented or thoroughly tested, and we are planning a replacement for this in #125.

I was trying some refactoring when trying #125 and this legacy blocked me.

### CODE REVIEW

File: lib/webidl2.js [hunk 1]
```diff
@@ -174,37 +174,14 @@
       "line-comment": /\/\/(.*)\r?\n?/y,
       "multiline-comment": /\/\*((?:[^*]|\*[^/])*)\*\//y
     };
-    function all_ws(store, pea) { // pea == post extended attribute, tpea = same for types
+    function all_ws(pea) { // pea == post extended attribute, tpea = same for types
```
Comment (marcoscaceres) [inline lib/webidl2.js]: At some point, we should consider renaming these things properly. There is no reason to have such short argument and variable names. Will make this code much more maintainable and approachable.

Comment (saschanaz) [reply lib/webidl2.js]: Ooops, actually that `pea` is also should be removed as it's part of opt.ws.

**Comment by saschanaz** [comment]
Should we add eslint to the CI so that we won't miss any unused variables?

**Comment by marcoscaceres** [comment]
> Should we add eslint to the CI so that we won't miss any unused variables?

Yeah, I think that would be good. When reviewing, it's hard to know if a variable gets used or not, because often parts of a function gets hidden.

**Pull request merged**